frontend/views/components/GroupSettings/InviteActionButton.vue

```diff
@@ -0,0 +1,60 @@
+<template lang="pug">
+menu-parent
+  menu-trigger.is-icon
+    i.icon-ellipsis-v
+
+  menu-content.c-content
+    ul
+      menu-item(
+        tag='router-link'
+        item-id='hash-3'
+        to='/somewhere'
+      )
+        i18n.icon-vote-yea.c-icon(tag='i') See original proposal
+      menu-item(
+        v-if='!linkExpired'
+        tag='router-link'
+        item-id='hash-3'
+        to='/somewhere'
```

Comment:
This won't ve a router-link, it will be just a button with @click, we can do that right away :)

Reply:
Fixed! :)